web/profiles/custom/yalesites_profile/modules/custom/ys_toolbar/src/ToolbarItemsService.php

```diff
@@ -235,4 +253,54 @@ protected function buildButton(string $route, string $label): array {
     ];
   }
 
+  /**
+   * Build an off canvas item for the toolbar.
+   *
+   * @param string $route
+   *   The route for the toolbar item destination.
+   * @param string $label
+   *   The text label for the toolbar item.
+   *
+   * @return array
+   *   A rennder array for a toolbar item.
+   */
+  protected function buildOffCanvasButton(string $route, string $label): array {
+    return [
+      '#type' => 'toolbar_item',
+      'tab' => [
+        '#type' => 'link',
+        '#title' => $label,
+        '#url' => Url::fromRoute(
+          $route,
+          $this->redirectDestination->getAsArray(),
+        ),
+        '#access' => $this->accessManager->checkNamedRoute(
+          $route,
+          $this->getNodeRouteParams(),
+          $this->account
+        ),
+        '#attributes' => [
```

Comment:
I was trying to find a way to disable the ajax loading spinner, but I do not see anything avilable within the toolbar_item render element. The spinning circle is distracting, but not a real problem. I will pitch this to the frontend folks for theming. Mentioning here in case you have any ideas.

Reply:
I disabled via CSS, just hid the spinner if it was attached to the toolbar items. I couldn't find another way to do it either - I tried a few things inside that service, but most of the other ways I found people doing this was for more regular form API render arrays, not the toolbar items.